src/main/scala/dxWDL/exec/TaskRunner.scala

```diff
@@ -17,13 +17,14 @@ task Add {
   */
 package dxWDL.exec
 
+import java.nio.file.{Files, Path, Paths}
+
 import com.dnanexus.DXAPI
 import com.fasterxml.jackson.databind.JsonNode
 import java.lang.management._
-import java.nio.file.{Files, Path, Paths}
 import spray.json._
-import wdlTools.eval.{Context => EvalContext, WdlValues}
-import wdlTools.types.{TypedAbstractSyntax => TAT, WdlTypes}
+import wdlTools.eval.{WdlValues, Context => EvalContext}
```

Comment:
The idea is to sort the names alphabetically, so `WdlValues` comes after `Context`.

Reply:
Fixed